data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -204,6 +204,19 @@ podman run -d --net host --privileged --name image-customization \
     --secret pull-secret,mode=400 \
     ${CUSTOMIZATION_IMAGE}
 
+# We're not interesting for the exit code, just that the server is available
+attempt=0
+until [ $attempt -eq 15 ] || curl -I "http://localhost:8084" > /dev/null; do
+    echo Waiting for image-customization controller to come up...
+    sleep 2
+    attempt=$(( $attempt + 1 ))
+done
+if [ $attempt -eq 15 ]; then
+    echo The image-customization controller did not come up in 30 seconds
+    podman logs image-customization
+    exit 1
```

Comment:
This will just restart the script I think? https://github.com/openshift/installer/blob/master/data/data/bootstrap/baremetal/systemd/units/ironic.service#L12

We really need a way to stop installation.

Using https://github.com/openshift/installer/blob/master/docs/dev/bootstrap_services.md#bootstrap-service-records looks like it would really help for debugging, but I'm not clear on whether anything in the installer will actually bail out once a failure is recorded. @staebler what is the recommended way of bailing out from a non-recoverable error?

Reply:
We can tell systemd to treat some exit codes as non-restartable `RestartPreventExitStatus=42` and use `exit 42` here. Will it work as a quick workaround?

Reply:
There is no current way for the bootstrap machine to inform the installer that it should bail out of waiting for the installation to complete.

Reply:
Thank you! Then, I think, aborting this script is the most obvious thing we can do.